Source: maxice8
Upstream: no
Reason: removes creation of fuzzing infrastructure which isn't included in the distfiles

--- meson.build
+++ meson.build
@@ -670,12 +670,6 @@
     message('valgrind not found, disabling valgrind test setup')
 endif
 
-
-# Fuzzing target programs.
-executable('fuzz-keymap', 'fuzz/keymap/target.c', dependencies: test_dep)
-executable('fuzz-compose', 'fuzz/compose/target.c', dependencies: test_dep)
-
-
 # Benchmarks.
 libxkbcommon_bench_internal = static_library(
     'xkbcommon-bench-internal',
